test/clamp.js

```diff
@@ -0,0 +1,24 @@
+
+var assert = require('assert');
```

Comment:
Let's replace this with `var eq = require('./shared/eq');`, to allow us to use `eq` in place of `assert.strictEqual`.